library/datasort.c

```diff
@@ -1128,6 +1128,108 @@ static void datasort_destroy(struct datasort_ctl *ds_ctl)
 	ds_ctl->chunks_dir = NULL;
 }
 
+/* Binlog entry mgmt subroutines */
+
+__attribute__ ((warn_unused_result))
+__attribute__ ((nonnull))
+inline struct eblob_binlog_entry *
+eblob_binlog_entry_new(const struct eblob_key *key)
```

Comment:
Why is it needed? How is it connected with this commit?

Reply:
returned it back